plugins/ptp_operator/ptp_operator_plugin.go

```diff
@@ -266,6 +265,7 @@ func Start(wg *sync.WaitGroup, configuration *common.SCConfiguration, fn func(e
 	log.Infof("setting up status listener")
 	for _, pType := range publishers {
 		baseURL := fmt.Sprintf(resourcePrefix, nodeName, string(pType.Resource))
+		log.Infof("for %s", fmt.Sprintf("%s/%s", baseURL, "status"))
```

Comment:
was this for debugging/verification only or do we need it? just want to make sure the log level is accurate..

Reply:
yes for verification only